platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -1087,11 +1109,18 @@ namespace platoon_strategic_ihp
                 //  -ensure plan type corresponds to join_index
 
                 // complete the request
-
-                int join_index = -1;
                 // UCLA: this is a newly added plan type 
+                std::string newLeaderPlatoonId = pm_.currentPlatoonID; // set self's (front join veh) platoon ID as the new platoon ID
```

Comment:
Hard to tell at this point if this is wrong, but I'm suspicious.  First, the host is imposing its own platoon ID on the existing platoon, which is different from how the front join is done above.  Second, if host is a solo joining vehicle, then its pm_.currentPlatoonID should be empty.  Third, the variable name and comments assume the joiner is doing a front join; however, this case 5 is supposed to cover joining in all 3 positions.

Reply:
I guess this is also a  left-over piece of code. It is safe to delete this line and the following line, as the variable "newLeaderPlatoonId" was only used twice through out this cpp file.  

I also double checked this method, the correct platoonId info was parsed from the mobility msg (the method's input), which is broadcasted by the target platoon leader. And the corresponding variable, the "platoonId", was used throughout this method (as it should) other than these two lines.

Reply:
removed